release/__init__.py

```diff
@@ -267,23 +267,36 @@ def get_gen_package_artifact(package_id_str):
         'local_path': package_filename}
 
 
-def make_bootstrap_artifacts(bootstrap_id, variant_name, artifact_prefix):
+def make_bootstrap_artifacts(bootstrap_id, package_ids, variant_name, artifact_prefix):
     bootstrap_filename = "{}.bootstrap.tar.xz".format(bootstrap_id)
+    bootstrap_filename = "{}.bootstrap.tar.xz".format(bootstrap_id)
+    active_filename = "{}.active.json".format(bootstrap_id)
+    active_local_path = artifact_prefix + '/bootstrap/' + active_filename
+    latest_filename = "{}bootstrap.latest".format(pkgpanda.util.variant_prefix(variant_name))
+    latest_complete_filename = "{}complete.latest.json".format(pkgpanda.util.variant_prefix(variant_name))
+
+    # Assert that the bootstrap active packages are in the package list.
+    with open(active_local_path) as f:
+        missing_packages = set(json.load(f)) - set(package_ids)
```

Comment:
A couple things here
1. Why do we have to do json loading here to get out hte data? Why can't we be passed it
2. can we use `pkgpanda.util.load_json` it makes nice error messages when things like the file doesn't exist or can't be parsed well as json
3. Rather than doing the set math here, and then assert on the length, if we just do an `assert set(active_packages) < set(package_ids)` py.test assertion introspection will do things like show a diff between the two sets. Can then leave a comment on the lin above as to what hte assert is doing (Adding an assert message disables py.test assert introspection)